src/erlazure_insights.erl

```diff
@@ -0,0 +1,132 @@
+-module(erlazure_insights).
+
+-export([
+        get_events/4,
+        get_time_offset_as_timestamp/2,
+        get_events_all/4,
+        follow_token/3
```

Comment:
probably this is the function to use if `get_events/4` returned some token?

I propose to stick to best practice for http rest api wrappers as follows:
1) 1 function per each http endpoint. (In our case - just one - which is `get_events`
2) That function should accept the same arguments as http endpoint (in our case - `subscription id, filters, selects, skipToken`)
2) If function works with continuation token - it should accept one (or possibly undefined for the first time)
3) For every continuation token let's introduce one more function for fetching them all, with `_all` postfix - `get_events_all` in our case
4) to simplify usage, some shortcuts can be introduced - in our case `Filter`(which is string) generator for [TS1, TS2] case seems to be very convenient